src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link
+  href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+  rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+  integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Movies App</title>
+</head>
+<body>
+  <header>
+    <nav class="navbar">
+      <a href="#" class="nav-logo">
+        <i class="fa-solid fa-film"></i>
+        <p>TV SHOW</p>
+      </a>
+      <ul class="menu-list">
+        <li class="list-item">
+          <a href="#">Home page</a>
```

Comment:
- [ ] Kindly display the total item on your homepage according to the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). 

>   <a href="#">Home page <span class="show-number">shows:</span></a>

Reply:
@Kweeka1 @Ndaboom Kindly ignore the video presentation required change. 🕺 Next reviewer should take note

> There is a separate activity for recording a video that would be assessed separately from the code.